### ISSUE TITLE
Rails 5 #tables deprecation warning when cached_tables: true

### ISSUE THREAD

**Issue by HarlemSquirrel** [DatabaseCleaner/database_cleaner#478]
I'm using version 1.5.3 of DatabaseCleaner with Rails 5.0.1 and receive the following warning when setting `DatabaseCleaner.strategy = :truncation, { cache_tables: false }`

```
DEPRECATION WARNING: #tables currently returns both tables and views. This behavior is deprecated and will be changed with Rails 5.1 to only return tables. Use #data_sources instead.
```

This is similar to the issue resolved with https://github.com/DatabaseCleaner/database_cleaner/pull/415

**Comment by voleoo** [comment]
I think it's fixed in master

**Comment by olleolleolle** [comment]
@HarlemSquirrel Does this happen with 1.6.1?

**Comment by HarlemSquirrel** [comment]
Everything seems to be fine with 1.5.3 @olleolleolle

**Comment by HarlemSquirrel** [comment]
@olleolleolle Things seems to be working well on 1.6.1 too.

**Comment by olleolleolle** [comment]
@HarlemSquirrel Is this issue then closable?

**Comment by etagwerker** [comment]
@olleolleolle @HarlemSquirrel I'm going to close this issue now. It's fixed starting in version 1.6.0: http://databasecleaner.github.io/2017/05/04/v1-6-0-is-out/

Thanks!
